core/src/services/icloud/core.rs

```diff
@@ -80,6 +82,8 @@ impl SessionData {
             session_token: None,
             scnt: None,
             account_country: None,
+
+            cookies: Default::default(),
```

Comment:
OpenDAL's code should be optimized for readability. Using `Default::default()` can make it difficult for readers to determine the type. Please use `BTreeMap::default()` instead.